internal/engine/engine.go

```diff
@@ -224,7 +224,7 @@ func (c *CraneEngine) ExecuteChecks(ctx context.Context) error {
 		checkElapsedTime := time.Since(checkStartTime)
 
 		if err != nil {
-			logger.WithValues("result", "ERROR", "err", err).Info("check completed", "check", check.Name())
+			logger.WithValues("result", "ERROR", "err", err.Error()).Info("check completed", "check", check.Name())
```

Comment:
Isn't this a change from `main`?

Reply:
Confirmed - Did something go sideways in a rebase?

Reply:
Actually, no. I think I had added that for something else. Won't hurt to leave it here for the moment. It will just get removed when we cherry-pick this back.